index.html

```diff
@@ -127,6 +131,17 @@
           </li>
         </ul>
       </header>
+
+      <section class="promo">
+        <div class="promo-text">the right ingredients for the right food</div>
+
+        <img src="../img/divider.png" alt="divider" />
+
+        <div class="promo-buttons">
+          <button class="btn btn-black">book a table</button>
+          <button class="btn btn-white">see the menu</button>
```

Comment:
Not critical, but a good manner
```suggestion
          <button class="btn btn-black">Book a table</button>
          <button class="btn btn-white">See the menu</button>
```